mysql-test/suite/galera/r/galera_FK_duplicate_client_insert,no_warnings.rdiff

```diff
@@ -0,0 +1,9 @@
+--- /home/jan/work/mariadb/10.11/mysql-test/suite/galera/r/galera_FK_duplicate_client_insert.result
++++ /home/jan/work/mariadb/10.11/mysql-test/suite/galera/r/galera_FK_duplicate_client_insert.reject
```

Comment:
Absolute path names do not add any useful information. Some other redundant information had been removed in 88d9348dfc494105046451beef15a8e68d7f51be.